### PR TITLE
Add weekly online gathering post

### PR DESCRIPTION
**Pull request by anadahz**

### CODE REVIEW

**Comment by francisco-core** [comment]
I did some minor changes (especially to the PT). See if you agree.

Only two things are missing:
  - on the metadata field for the date, you can also specify the time (we should do that).
  - this is an event so it should be moved from `/post/` to `/events`, no?

File: content/post/weekly-discussion.en.md [hunk 1]
```diff
@@ -12,19 +12,20 @@ Dear **PrivacyLx**ers,
...
 
-The first gathering will take place on **Wednesday, 11 November, 2020**.
+Join us every wednesday at 7pm (Lisbon time). The first gathering will take place this **Wednesday, 11 November, 2020**.
+
 Save the date and join us!
 
 ---
 
-**When:** Every Wednesday at 19:00\
+**When:** Every Wednesday at 7pm\
```
Comment (anadahz) [inline content/post/weekly-discussion.en.md:28]: People may get confused with `pm` instead of the 24 hour time format.

**Comment by anadahz** [comment]
@francisco-core thanks for reviewing the PR.

> on the metadata field for the date, you can also specify the time (we should do that).

I added the time in both posts.

> this is an event so it should be moved from /post/ to /events, no?

Currently the events section is made to provide a one time "event". Given that the gathering will be happening weekly, we will need to present in a different way to the website, until then I guess a post is OK.
Does this makes sense?

File: content/post/weekly-discussion.en.md (hunk 1 above)
Comment (francisco-core) [reply content/post/weekly-discussion.en.md]: ok

**Comment by francisco-core** [comment]
> > this is an event so it should be moved from /post/ to /events, no?
> 
> Currently the events section is made to provide a one time "event". Given that the gathering will be happening weekly, we will need to present in a different way to the website, until then I guess a post is OK.
> Does this makes sense?

Ok. I thought the same. It's just that when you click on "events" it won't show and when we post another one it's not gonna show. By making it in `/events` we'll have to update it weekly either way. So it'll be annoying either way.

We can leave it as a `/post`.

Maybe another idea is to later add a comment to https://privacylx.org/community/ about this meeting.

File: content/post/weekly-discussion.en.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "Community Gatherings on Wednesdays"
 author: "PrivacyLx"
-date: "2020-11-09T15:00:00Z"
+date: "2020-11-09T19:00:00Z"
```
Comment (anadahz) [inline content/post/weekly-discussion.en.md:4]: This is not the date of the event but the date of the post.

**Comment by anadahz** [comment]
>Maybe another idea is to later add a comment to https://privacylx.org/community/ about this meeting.

I was thinking the same, we can add a box on the community page mentioning the weekly gathering.

**Comment by francisco-core** [comment]
let's merge this

**Pull request merged**